tests/specs/graph/JsrSpecifiers_FastCheck_Enums.txt

```diff
@@ -108,3 +120,50 @@ Fast check https://jsr.io/@scope/a/1.0.0/mod.ts:
   enum NonExportedEnum {
     Value
   }
+  const NUM: any = {} as any;
+  export enum Foo1 {
+    A = 1,
+    B = "2",
+    C = 1 << 2,
+    D = 1 + 2,
+    F = Foo1.A | Foo1.C,
+    G = Foo1.A + Foo1.C,
+    H = new Public1().test + 1,
+    I = 1 + NUM
+  }
+  --- DTS ---
+  export declare enum EnumWithNoInits {
+    Value1,
+    Value2
+  }
+  export declare enum EnumWithNumInits {
+    Value1 = 1,
+    Value2 = 2
+  }
+  export declare enum EnumWithStringInits {
+    Value1 = "a",
+    Value2 = "b"
+  }
+  declare const value: number;
+  export declare enum EnumWithNonConstInits {
+    Value1,
+    Value2
+  }
+  declare class Public1 {
+  }
+  declare class Public2 {
+  }
+  declare enum NonExportedEnum {
+    Value
+  }
+  declare const NUM: any;
+  export declare enum Foo1 {
+    A = 1,
+    B = "2",
+    C = 1 << 2,
+    D = 1 + 2,
+    F = Foo1.A | Foo1.C,
+    G = Foo1.A + Foo1.C,
+    H,
+    I = 1 + NUM
```

Comment:
It seems this is also not allowed in dts files.

Reply:
Yeah wanted to add a couple cases where TS strips the type but still generates the files even with a type error.

Reply:
We probably shouldn't follow TS here and instead just strip this with a warning diagnostic.